contract_interface.md

```diff
@@ -9,6 +9,7 @@ Every contract must be stored in the [WebAssembly Binary Encoding](https://githu
 ### Imports
 
 A contract can only import symbols specified in the [Ethereum Environment Interface](./eth_interface.md).
+In practice, this means that all imports specified by an ewasm module must be from the `ethereum` namespace, and having a function signature and name directly correspondent to a function specified in the EEI.
```

Comment:
We have a `debug` namespace too

Reply:
On-chain code does not, but I will make not anyway

Reply:
Fixed in #116.